src/scripting/lua_menu_item.cpp

```diff
@@ -24,26 +24,84 @@ static lg::log_domain log_scripting_lua("scripting/lua");
 #define LOG_LUA LOG_STREAM(info, log_scripting_lua)
 #define ERR_LUA LOG_STREAM(err, log_scripting_lua)
 
+static const char menuitemKey[] = "menu_item";
```

Comment:
I think changing the underscore to a space would be more in line with other metatable names we use. I'm not aware of any that use an underscore for multi-word names.